packages/cli/src/environments.ee/source-control/resource-handlers/generic-resource-handler.ts

```diff
@@ -0,0 +1,56 @@
+import type { SourceControlledFile } from '@n8n/api-types';
+import { Service } from '@n8n/di';
+
+@Service()
+export class GenericResourceHandler {
```

Comment:
I can't see where this service is being used. Can you point me to it @guillaumejacquart ?

Reply:
Ah yes I see it now.
What is the motivation behind this being called the `GenericResourceHandler`? Is there a realistic need for other types of resource handlers in the future?

When I first read it i wondered "what makes this generic", seeing that it's quite specific to source controlled files.

Reply:
Yeah I think you're right, does not make sense. this is actually rather just a "helper" function to manipulate file
I might remove this file all together and move the functions to the source-control-helper file, what do you think ?

Reply:
That `source-control-helper.ee.ts` is quite larger already. I think renaming this class to `SourceControlResourceHelper` would already be fine.
Or exporting the functions in a separate `source-control-resource-helpers.ts` as it seems it does not need to be injectable.